Source/Style.js

```diff
@@ -0,0 +1,124 @@
+// ********************************
+// # Style.js
+// # Sketch Javascript API.
+//
+// All code (C) 2016 Bohemian Coding.
+// ********************************
+
+import { WrappedObject } from './WrappedObject.js'
+
+/// A solid fill/border.
+const BCFillTypeColor    = 0
+
+/// A gradient fill/border.
+const BCFillTypeGradient = 1
+
```

Comment:
I guess 2 and 3 are Radial and Angular gradients? Are they missing for any particular reason?